src/main/app/claims/claimStoreClient.ts

```diff
@@ -105,10 +104,8 @@ export class ClaimStoreClient {
         }
       })
       .then(claim => {
-        if (!FeatureToggles.isEnabled('ccd')) { // CCD does authorisation checks for us
-          if (user.id !== claim.submitterId && user.id !== claim.defendantId) {
-            throw new ForbiddenError()
-          }
+        if (user.id !== claim.submitterId && user.id !== claim.defendantId) {
```

Comment:
@jacmurphy `claim.submitterId` does not exist. Please add types to detect that issue at compile time.